### PR TITLE
feat(dashboard): send 'github_app_authorization' to the dahsboard

### PR DESCRIPTION
**Pull request by GuillaumeOj**

### CODE REVIEW

File: mergify_engine/github_events.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ def get_ignore_reason(event_type, data):
         "check_suite",
         "check_run",
         "refresh",
+        "github_app_authorization",
```
Comment (jd) [inline mergify_engine/github_events.py:77]: I don't think the engine is going to handle those events, so it's fine it returns "unexpected"

**Review by jd — CHANGES_REQUESTED** [review]
This is not going to forward them

File: mergify_engine/github_events.py (hunk 1 above)
Comment (GuillaumeOj) [reply mergify_engine/github_events.py:77]: Ok, i'm going to look elsewhere

**Comment by GuillaumeOj** [comment]
No change to be down on the engine, we just need to update the env right env variables: `WEBHOOK_FORWARD_EVENT_TYPES`

**Pull request closed without merging**